src/pkg/selectors/selectors_test.go

```diff
@@ -239,3 +240,126 @@ func (suite *SelectorSuite) TestSplitByResourceOnwer() {
 		})
 	}
 }
+
+// TestPathCategoriesAll includes all the included selectors do not return `Unknown` Category
+func (suite *SelectorSuite) TestPathCategoriesAll() {
+	users := []string{tester.LoadTestM365UserID(suite.T())}
```

Comment:
It's not necessary to load a user, since none of these values are checked against the tenant.  I'd suggest using a static string instead.

Reply:
Resolved within [fa9b7ad](https://github.com/alcionai/corso/pull/2111/commits/fa9b7ad72227a7b60a2fe1532d5112eb3a430654)